model/import/AssemblyImportFailedException.php

```diff
@@ -0,0 +1,29 @@
+<?php
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2019 (original work) Open Assessment Technologies SA ;
+ */
+
+namespace oat\taoDeliveryRdf\model\import;
+
+
+class AssemblyImportFailedException extends \common_exception_ClientException
+{
+    public function getUserMessage()
+    {
+        return $this->getMessage();
```

Comment:
the split between messages that can be send to the end user and internal error messages is intended to prevent leaks of critical system information and to ensure the end user messages are translated. This would bypass that mechanism

Reply:
This message extends `common_exception_ClientException` and I created it to make sure that end user (administrator who has access to run CLI script) will have original error message instead of generic one.